pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-signald/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, pidgin, json-glib, signald }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-signald";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "hoehermann";
+    repo = "libpurple-signald";
+    rev = "v${version}";
+    sha256 = "sha256-2LiHjVRBwdPbfravIVM+gvsh3Gq4bhjtRD6eWAbkWmc=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    pidgin
+    json-glib
+    signald
+  ];
+
+  PKG_CONFIG_PURPLE_PLUGINDIR = "${placeholder "out"}/lib/purple-2";
+  PKG_CONFIG_PURPLE_DATADIR = "${placeholder "out"}/share";
+
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/hoehermann/libpurple-signald";
+    description = "Signald plugin for Pidgin";
```

It seems like a lot of our Pidgin / libpurple plugins use a description like this (took this format from `purple-matrix`). Probably would be best to mention both Pidgin and libpurple so people can more easily find this when searching for packages.

```suggestion
    description = "Signal support for Pidgin / libpurple";
```

---

Thank you for the feedback!